scripts/main.js

```diff
@@ -0,0 +1,50 @@
+const btnSubmit = document.querySelector('.submit');
+
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
+
+let bookid = 0;
+
+function appendDiv(book) {
+  let titleElement = ' ';
+  titleElement += `
+            <div class="book" data-id="${book.id}">
+              <h2 class="title">${book.title}
+                  <span class="author">${book.author}</span>
+               </h2>
+              <button class="btn btn-remove">Remove</button>
+              <hr/>
+            </div>
+          `;
+
+  const sectionBookDisplay = document.querySelector('.book-display');
+  sectionBookDisplay.innerHTML += titleElement;
+}
+
+btnSubmit.addEventListener('click', () => {
+  const title = document.querySelector('.form-title').value;
+  const author = document.querySelector('.form-author').value;
+
+  bookid += 1;
+
+  const book = new Book(title, author, bookid);
+
+  appendDiv(book);
+});
+
+const bookDisplay = document.querySelector('.book-display');
+
+bookDisplay.addEventListener(
+  'click',
+  (e) => {
+    if (e.target.tagName === 'BUTTON') {
+      const currentDiv = e.target.parentElement;
+      currentDiv.parentElement.removeChild(currentDiv);
+    }
+  }, true,
+);
```

Comment:
- [ ] Please make sure to implement the add and remove functions outside of the event listeners and call them inside of the event listeners, instead of writing the implementation directly into the event listeners. Following JS best practices and the "SOLID" principle, that would separate your code into pieces that don't affect one another directly.